### DISCUSSION TITLE
Support `--version` option

### DISCUSSION THREAD

**Discussion by vladdeSV** [jotaen/klog#374] [Feature ideas & discussions]
The option `--version` is common in many other tools (`git --version`, `brew --version`, `wget --version`, etc.).

It is so common that I instinctively run `klog --version` when I want to find out the version of *klog* I am using. However, `--version` is not a valid option, and instead just the action `version` should be used.

I suggest allowing `klog --version` to be run, which would act the same way as `klog version` does today.

**Comment by vladdeSV** [comment]
Out of all the available actions *klog* supports, `version` is the only one which I believe belongs as an option for the base executable `klog`.

**Comment by jotaen** [comment]
As far as I remember there was some caveat with the CLI library I’m using. I’m not exactly sure anymore what the issue was, however. Generally, I agree it’s more intuitive to support `klog --version` / ~`klog -v`~

**Comment by vladdeSV** [comment]
Sounds great. One point of notice, and I am not sure if this applies here, but `-v` sometimes is used for verbose output (and the more `v`s you add, the more vebose, eg. `-vvv`)

**Comment by jotaen** [comment]
Good point, I think `--version` should be enough then.
